demos/showcase/src/app/app.component.ts

```diff
@@ -85,4 +87,20 @@ export class AppComponent {
     openDrawer(): void {
         this._stateService.setDrawerOpen(true);
     }
+
+    toggleTheme(): void {
+        const body = document.querySelector('body') as HTMLElement;
+        if (this.isDarkMode) {
+            body.classList.remove('pxb-blue-dark');
+            body.classList.add('pxb-blue');
+        } else {
+            body.classList.remove('pxb-blue');
+            body.classList.add('pxb-blue-dark');
+        }
+        this.isDarkMode = !this.isDarkMode;
+    }
+
+    toggleDirectionality(): void {
+        this.isRtl = !this.isRtl;
```

Comment:
set `dir` on body tag?

Reply:
I'm still keeping isRtl for convenient inline rtl style rules; there's a few of them that were recently added.

Reply:
Yeah, I didn't mean to replace it...I was thinking more of an 'in addition to'